src/components/views/dialogs/InfoDialog.js

```diff
@@ -28,12 +28,14 @@ export default React.createClass({
         description: PropTypes.node,
         button: PropTypes.string,
         onFinished: PropTypes.func,
+        hasCloseX: PropTypes.bool,
```

Comment:
tbh I think I'd prefer `hasCloseButton`

Reply:
I think `hasCancel` would match what we have elsewhere, for better or worse.

Reply:
That was my original wording but then it left ambiguity as two components within InfoDialog have a `hasCancel` prop, the dialog itself (for the closing X) and DialogButtons, so it'd be unclear which/both it affected

Reply:
*everything is awful themesong plays*

as much as I normally argue for consistency, `hasCloseButton` is probably safer given @t3chguy's comment imo.